.rubocop.yml

```diff
@@ -0,0 +1,47 @@
+# The behavior of RuboCop can be controlled via the .rubocop.yml
+# configuration file. It makes it possible to enable/disable
+# certain cops (checks) and to alter their behavior if they accept
+# any parameters. The file can be placed either in your home
+# directory or in some project directory.
+#
+# RuboCop will start looking for the configuration file in the directory
+# where the inspected file is and continue its way up to the root directory.
+#
+# See https://docs.rubocop.org/rubocop/configuration
+
+AllCops:
+  NewCops: enable
+  TargetRubyVersion: 3.0
+
+Layout/HashAlignment:
+  EnforcedHashRocketStyle:
+    - separator
+  EnforcedColonStyle:
+    - separator
+
+Layout/FirstHashElementIndentation:
+  EnforcedStyle: consistent
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/Documentation:
+  Enabled: false
+
+Naming/VariableNumber:
+  Enabled: false
+  
+Naming/ClassAndModuleCamelCase:
+  Enabled: false
+
+# ncurses.rb:11:3: C: Metrics/MethodLength: Method has too many lines. [31/10]
+#  def self.build ...
+#  ^^^^^^^^^^^^^^
+Metrics/MethodLength:
```

Comment:
Any way to disable for just the one file or method instead of globally, or if there is a sane higher limit maybe apply that instead?

Reply:
https://docs.rubocop.org/rubocop/configuration.html#includingexcluding-files
https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code